Chocolatey.Cake.Recipe/Content/parameters.cake

```diff
@@ -438,6 +441,19 @@ public static class BuildParameters
         ShouldReportCodeCoverageMetrics = shouldReportCodeCoverageMetrics;
         ShouldReportUnitTestResults = shouldReportUnitTestResults;
         ShouldRunChocolatey = shouldRunChocolatey;
+
```

Comment:
I think we are missing a line here:

```
ShouldRunDependencyCheck = shouldRunDependencyCheck;
```

To accept the value passed in from the recipe.cake file, whilst allowing it to be overridden by the command line argument.

Reply:
I was following the example of SonarQube here, should I also add a `ShouldRunSonarQube = shouldRunSonarQube;` while I'm in there?

n.b. this has been rebased, so the line numbers have changed:
* DependencyCheck is now on line: 662
* SonarQube is now on line: 758